nodejs/week1/warmup.js

```diff
@@ -0,0 +1,21 @@
+class Circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+  getDiameter() {
+    const diameter = this.radius * 2;
+    return diameter;
+  }
+  getCircumference() {
+    const circumference = 2 * Math.PI * this.radius;
+    return circumference;
+  }
+  getArea() {
+    const area = Math.PI * this.radius * this.radius;
```

Comment:
here you could also use Math.pow() if you want to learn a method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow - but it's just a matter of preference :)

Reply:
thanks